angular-frontend/src/app/omnai-datasource/omnai-scope-server/live-data.service.ts

```diff
@@ -60,25 +64,37 @@ export class OmnAIScopeDataService implements DataSource{
 
   readonly #httpClient = inject(HttpClient);
 
+  private getAvailableDevices(): void {
```

Comment:
If a function is called `get`sth, I would expect it to return whatever it should get. As this is a setup function, I would name it that way
```suggestion
  private setupDevicePolling(): void {
```